src/chi/components/date-picker/date-picker.scss

```diff
@@ -20,6 +20,13 @@ $mondayStartingWeek: (
   sun: 6
 );
 
+.a-inputWrapper.-disabled {
+  pointer-events: none;
+  & .a-input, & .a-icon {
```

Comment:
@janibolkvadze .a-input and .a-icon should follow the look and feel of disabled inputs. Please remove this property.